task1:
  ros__parameters:
    no_of_shelf: 20
    shelf_init_pos_row: [3,3,4,4,5,5,6,6,7,7,3,3,4,4,5,5,6,6,7,7]
    shelf_init_pos_col: [2,3,2,3,2,3,2,3,2,3,5,6,5,6,5,6,5,6,5,6]
    no_of_picking_stations: 4 
    pos_of_picking_stations_row: [1,1,9,9]
    pos_of_picking_stations_col: [2,6,2,6]
